conf/messages

```diff
@@ -1366,20 +1368,30 @@ ated.delete-draft.title = Are you sure you want to delete this draft return?
 ated.delete-draft.error = delete draft return
 ated.delete-drafts.error.general.yesNo = Select if you want to delete this draft or not
 yes-no.error.mandatory.delete.draft = You must answer this question
-yes-no.error.mandatory.existing.returns = The existing return question must be answered
+yes-no.error.mandatory.existing.returns = Select yes if you filed a return for this property last year
```

Comment:
This message already exists L1384. Not sure if the content should be under the DELETE DRAFT RETURN section anyway?